tests/components/sensor/test_darksky.py

```diff
@@ -138,9 +137,7 @@ def test_setup_bad_api_key(self, mock_get_forecast):
         msg = '400 Client Error: Bad Request for url: {}'.format(url)
         mock_get_forecast.side_effect = HTTPError(msg,)
 
-        response = darksky.setup_platform(self.hass, VALID_CONFIG_MINIMAL,
-                                          MagicMock())
-        assert not response
+        assert setup_component(self.hass, 'sensor', VALID_CONFIG_MINIMAL)
```

Comment:
Not sure this is the same test... the original expects setup_platform to fail

Reply:
You are correct, this is testing something different now. Fixed.